release-notes/5.16.4.md

```diff
@@ -0,0 +1,44 @@
+# CiviCRM 5.16.4
+
+Released September 3, 2019
+
+- **[Synopsis](#synopsis)**
+- **[Bugs resolved](#bugs)**
+- **[Credits](#credits)**
+- **[Feedback](#feedback)**
+
+## <a name="synopsis"></a>Synopsis
+
+| *Does this version...?*                                         |         |
+|:--------------------------------------------------------------- |:-------:|
+| Fix security vulnerabilities?                                   |   no    |
+| Change the database schema?                                     |   no    |
+| Alter the API?                                                  |   no    |
+| **Require attention to configuration options?**                 | **yes** |
+| Fix problems installing or upgrading to a previous version?     |   no    |
+| Introduce features?                                             |   no    |
+| **Fix bugs?**                                                   | **yes** |
+
+## <a name="bugs"></a>Bugs resolved
+
+- **_Smart Groups_: Relative date filters saved incorrectly ([dev/core#1226](https://lab.civicrm.org/dev/core/issues/1226):
+  [#15201](https://github.com/civicrm/civicrm-core/pull/15201), [#15169](https://github.com/civicrm/civicrm-core/pull/15169))**
+
+  On 5.16.0 - 5.16.3, if you saved a smart-group with a relative date
```

Comment:
@totten maybe mention that it was specific fields maybe?

Reply:
Good idea - but the list is a bit long, so I'll copy it into the main description of https://lab.civicrm.org/dev/core/issues/1226